python/setup.py

```diff
@@ -245,6 +243,11 @@ def build_extension(self, ext):
         "Topic :: Software Development :: Build Tools",
         "License :: OSI Approved :: MIT License",
         "Programming Language :: Python :: 3.6",
```

Comment:
Python 3.6 reached the end of its support schedule 1 year and 3 months ago (23 Dec 2021).

Also, there are `from __future__ import annotations` imports (added in Python 3.7) in the codebase. The code is not able to run with Python 3.6 already.

https://github.com/openai/triton/blob/46672772b46b103db7341c9e10fbad7f643557d4/python/triton/compiler.py#L1

```console
$ python3.6 -c 'from __future__ import annotations'
  File "<string>", line 1
SyntaxError: future feature annotations is not defined
```